scripts/system/checkForUpdates.js

```diff
@@ -0,0 +1,54 @@
+//  checkForUpdates.js
+//
+//  Created by Kalila L. on 8/27/20
+//  Copyright 2020 Vircadia contributors.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+Script.include("/~/system/libraries/utils.js");
+var networkingConstants = Script.require("./libraries/networkingConstants.js");
+
+// We'll want a way to specify this later, auto detection can also be a thing, but strictly optional.
+var currentBuildType = "latest";
+var currentVersion = "2020.2.3-Asteria"; // HifiAbout.buildVersion; 
```

Comment:
Rather than hard-coding this in - requiring an update each release - how about using the values from the `HifiAbout` API?

Reply:
Yup! That's commented out precisely for testing, the commit says to "revert" it once we're ready to merge. :)